pkg/chunkenc/varint.go

```diff
@@ -0,0 +1,123 @@
+package chunkenc
+
+import "bufio"
+
+// unrolledDecodeUVarint decodes a unsigned varint from the provided buffer.
+// This is the same as the std binary library except that it avoids a for loop.
```

Comment:
Could you link the original? Is it https://cs.opensource.google/go/go/+/master:src/encoding/binary/varint.go?

Reply:
Good idea will do.